sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -911,9 +911,9 @@ public boolean requiresDeduping() {
     @VisibleForTesting
     static final String METRIC_NAMESPACE = "KafkaIOReader";
     @VisibleForTesting
-    static final String CHECKPOINT_MARK_COMMITS_METRIC = "checkpointMarkCommits";
-    private static final String CHECKPOINT_MARK_COMMIT_SKIPS_METRIC = "checkpointMarkCommitSkips";
-
+    static final String CHECKPOINT_MARK_COMMITS_ENQUEUED_METRIC = "checkpointMarkCommitsEnqueued";
```

Comment:
prefer to keep the same name if it's not ambiguous。

Reply:
Commented on related issue below. 'Enqueued' name might make more sense.